lib/rubocop/cop/rails/after_commit_override.rb

```diff
@@ -78,6 +78,13 @@ def class_send_nodes(class_node)
         def after_commit_callback?(node)
           AFTER_COMMIT_CALLBACKS.include?(node.method_name)
         end
+
+        def named_callback?(node)
+          name = node.first_argument
+          return false unless name
+
+          name.sym_type?
```

Comment:
I guessed it also needs `str_type?` check, but it looks unnecessary.
I tried to define a callback with a string name, like `after_commit 'foo'` in a Rails app, but it raises an error during defining the callback. So we only need `sym_type?` check.